### PR TITLE
Renamed variables that shadowed builtins

### PR DESCRIPTION
**Pull request by ppajersk**
Merged if elif statements that contained idenical implementations
Removed identical def code block
Removed unused local variables

### CODE REVIEW

File: vmanage/api/policy_updates.py [hunk 1]
```diff
@@ -132,13 +132,7 @@ def update_policy_definition(self, policy_type, name, policy_id, policy_def, new
...
         #pylint: disable=too-many-nested-blocks
         for item1 in policy_def["sequences"]:
-            if item1["sequenceName"] == seq_name:
-                for item2 in item1["actions"]:
-                    if item2['type'] == 'slaClass':
-                        for item3 in item2['parameter']:
-                            if item3["field"] == 'preferredColor':
-                                item3["value"] = new_color
-            elif seq_name is None:
+            if item1["sequenceName"] == seq_name or seq_name is None:
```
Comment (jlothian) [inline vmanage/api/policy_updates.py:135]: Not sure you wanted to delete this entire code block?

**Review by jlothian — CHANGES_REQUESTED** [review]

File: vmanage/api/policy_updates.py (hunk 1 above)
Comment (ppajersk) [reply vmanage/api/policy_updates.py:135]: Its identical to lines 142-146. I also combined the if elif statements on line 135 and 141.

Comment (jlothian) [reply vmanage/api/policy_updates.py:135]: Ah got it, missed that bit.

Comment (ppajersk) [reply vmanage/api/policy_updates.py:135]: Should I resolve this conversation.

**Review by jasonking3 — APPROVED** [review]
LGTM

**Pull request merged**